vqt/common.py

```diff
@@ -36,6 +36,7 @@ def __call__(self):
         except Exception as e:
             logger.warning("error in ACT(%s, %s, %s)", str(self.meth), str(self.args), str(self.kwargs))
             logger.warning(str(e))
+            logger.info("", exc_info=1)
```

Comment:
If we're going to log the exception trace, probably best to just get rid of the `logger.warning(str(e))` and throw the exc_info onto the `error in ACT` log message.

Reply:
i wanted to leave the stack trace for "info" while having the limited info for "warning".  i still prefer what's there.  this keeps a standard logging level from having too much log vomit, but makes it available without changing the code, making it accessible to all of our users with a simple commandline -vvv.